tests/vmi_networking_test.go

```diff
@@ -831,9 +831,6 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 			})
 
 			table.DescribeTable("should have the correct MTU", func(ipFamily k8sv1.IPFamily) {
-				if ipFamily == k8sv1.IPv6Protocol {
-					libnet.SkipWhenNotDualStackCluster(virtClient)
```

Comment:
Why do you remove it? How would it work on single stack clusters?

Reply:
Done, the needed coded has been backported too.